chaoslib/litmus/platform/gke/node-cpu-hog-job.j2

```diff
@@ -1,15 +1,24 @@
 apiVersion: batch/v1
 kind: Job
 metadata:
+{% if c_engine is defined and c_engine != '' %}
   name: node-cpu-hog-{{ chaos_uid }}
```

Comment:
Can you please chaos_uid to run_id? and generate run_id before calling this jinja template. Please refer `chaoslib/pumba/network_chaos/pumba_netem_job.j2`